x-pack/plugins/security_solution/public/timelines/components/timeline/body/unified_timeline_body.test.tsx

```diff
@@ -57,39 +53,6 @@ describe('UnifiedTimelineBody', () => {
   beforeEach(() => {
     (UnifiedTimeline as unknown as jest.Mock).mockImplementation(MockUnifiedTimelineComponent);
   });
-  it('should pass correct page rows', () => {
```

Comment:
Responsibility of collating events has been moved `useTimelineEvents` so this test is not needed here.